nicesite.so.hosting.json

```diff
@@ -0,0 +1,24 @@
+{
+  "providerId": "nicesite.so",
+  "providerName": "nicesite",
+  "serviceId": "website",
+  "serviceName": "nicesite",
+  "version": 1,
+  "logoUrl": "https://ucarecdn.com/541766e8-3c80-41b2-b3ad-5a3c3da88806/nicesite_logo_circlecrop.png",
+  "description": "Enables web hosting for nicesite domains.",
+  "variableDescription": "v1 is the apex domain",
+  "records": [
+    {
+      "type":"A",
+      "host":"@",
+      "pointsTo":"75.2.60.5",
+      "ttl": 60
```

Comment:
Sure about such short TTL? 3600 is more typical for a website. It may negatively affect performance of DNS resolution.
It's just a remark, this is not blocking template merge.

Reply:
I updated to 300, which is same as Webflow's- we're hoping the changes are speedier than 3600.